test/integration/openshift_health_checker/preflight/preflight_test.go

```diff
@@ -0,0 +1,95 @@
+package preflight
+
+import (
+	"testing"
+
+	. ".."
```

Comment:
FTR, this is something I wrote and I don't like it; but to use absolute imports we need to have the code checked out appropriately within a GOPATH.